components/card/card--small.readme.md

```diff
@@ -0,0 +1,5 @@
+- Small sized cards can contain images with 16:9, 3:2 or 1:1 aspect ratios.
```

Comment:
**Suggestion (non-blocking):** could we probably move these notes into the `card.config.yml` file? There is already a `readme.md` file for when you click on the Card item from the sidemenu, and it would probably be more neat to have these notes also within the YAML file?